examples/demos/utils.go

```diff
@@ -28,14 +29,19 @@ func makeUntypedData(v interface{}) untypedData {
 	return untypedData{V: v}
 }
 
+// TODO: test this file thoroughly. Previous code didn't panic so ok should be true.
```

Comment:
I'm having trouble understanding this code but since it didn't panic before it won't be broken now.

Reply:
Previous type assertion could panic in runtime, current one never panics. I don't see a reason for that TODO.

Reply:
Done